authors/apps/article_tags/tests/base.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.test import APIClient, APITestCase
+from django.urls import reverse
+# Local import
+from .test_data import TestData
+from authors.apps.authentication.models import User
+
+
+class BaseTestCase(APITestCase):
```

Comment:
some thing here. please add a description for the class

Reply:
I have added a doc string.. Thanks for the observation